lib/ansible/module_utils/aws/elb_utils.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+# Copyright (c) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# Ansible imports
+from ansible.module_utils.ec2 import camel_dict_to_snake_dict
+
+# Non-ansible imports
+try:
+    from botocore.exceptions import ClientError
+except ImportError:
+    pass
+import traceback
+
+
+def get_elb(connection, module, elb_name):
+    """
+    Get an ELB based on name. If not found, return None.
+
+    :param connection: AWS boto3 elbv2 connection
+    :param module: Ansible module
+    :param elb_name: Name of load balancer to get
+    :return: boto3 ELB dict or None if not found
+    """
+
+    try:
+        load_balancer_paginator = connection.get_paginator('describe_load_balancers')
+        return (load_balancer_paginator.paginate(Names=[elb_name]).build_full_result())['LoadBalancers'][0]
+    except ClientError as e:
+        if e.response['Error']['Code'] == 'LoadBalancerNotFound':
+            return None
+        else:
+            module.fail_json(msg=e.message, exception=traceback.format_exc(), **camel_dict_to_snake_dict(e.response))
```

Comment:
Python3 does not have exception.message. Instead, use a custom message to describe the failed operation and add the exception to the end, like
`module.fail_json(msg='Unable to describe load balancers: {0}'.format(to_native(e)), ...)`

Need to remove e.message in other places too.